### ISSUE TITLE
Add a `--overwrite-if-different`

### ISSUE THREAD

**Issue by scarlehoff** [NNPDF/pineko#207]
There are certain situations where we might not be sure whether an eko can be reused or not (for instance, if one tries to create both theories 40_000_000 and 41_000_000, many ekos will be the same for NLO and NNLO, but some won't).

The clever user might copy all ekos from one folder into the other, thinking they are hacking the system... however all sort of problems will arise.
Ideally `pineko` will look at the theory and operator inside the eko and realize when the resulting product would be exactly the same and when it won't.

**Comment by felixhekhorn** [comment]
mh? how would that work? you copy the ekos in any case and then generate the op card and then check the new generated against the one inside the eko? is this what you are suggesting?

just to remind you that `pineko` has the feature of inheriting, i.e. no need to copy - linking is sufficient

we were briefly thinking in the past about checking if two (NNPDF) theories are compatible (eko- or grid-wise), but then decided that the check would be too complex ...

**Comment by scarlehoff** [comment]
> generate the op card and then check the new generated against the one inside the eko? is this what you are suggesting?

Yes. Exactly this. If the theory and operator card are the same, the eko will be.

> just to remind you that pineko has the feature of inheriting, i.e. no need to copy - linking is sufficient

Yes, yes, I do link. I mean, not me me, the clever anoymous user.

**Comment by felixhekhorn** [comment]
> > generate the op card and then check the new generated against the one inside the eko? is this what you are suggesting?
> 
> Yes. Exactly this. If the theory and operator card inside the eko are the same, the eko will be.

I guess that should be doable ... however, I'd say let's restrict to bit-by-bit-equality (else it will be complicated again) and this would neglect different card version

of course for that we need the opcard - cc #202

**Comment by scarlehoff** [comment]
The operator card is always inside the eko and needs to be generated even if not written down to create the new eko.

But anyway, in #202 you already convinced me about keeping the opcard and removing only the template.
